_posts/2019-11-05-20Year-Em-Tech-Landscape.md

```diff
@@ -0,0 +1,67 @@
+
+The landscape can be found here. 
+A detailed description of the Emerging Technology Landscape can be found here.
+
+### How the Landscape was constructed
+The selected technologies and their positioning on the landscape has been informed by research and shaped by consultation with other emerging tech leaders engaged through participation in forums.  The landscape was constructed by applying a governmental lens which enables a better digital public service.
+These sources include:
+
+- [OECD](https://www.oecd.org/sti/emerging-tech/) 
+- [World Economic Forum](https://www.weforum.org/agenda/2019/07/these-are-the-top-10-emerging-technologies-of-2019/)
+- [McKinsey Institute](https://www.mckinsey.com/business-functions/mckinsey-digital/our-insights/disruptive-technologies)
+- [Gartner - Hype Cycle](https://www.gartner.com/en/documents/3887767)
+- [SAP](https://www.sap.com/australia/trends.html)
+- [AIForum New Zealand](https://aiforum.org.nz/)
+
+### Technology to focus on
+With such a vast array of technology in the landscape the lab has seven key areas of technology that should have a high priority focus.
+- Artificial Intelligence
+- Spatial Computing
+- Encryption
+- Internet of Things
+- Big Data
+- Robotics and Autonomous Systems
+- Quantum Computing
+
+## Opportunities for Further Work
+This is a piece of work that has the potential to expand and develop. 
+Possible future work could be:
+
+**Application of different lenses**
+Applying different lenses onto this landscape will most likely see the landscape change. An example of another lens would be that of digital inclusion, which would identify technology that could ensure a more equitable digital environment for all. 
+
+**Workshops with external partners** 
+To help broaden the technologies represented in the landscape engaging with both other government agencies and various private sector organisations would she light into what developing trends there are within New Zealand. 
+
+**Landscape visualisation development** 
+The landscape can be developed to be more interactive and more interconnected. Development of interactive cards of the landscape which display definitions of the technology could be developed. 
+Furthermore development of visualisation connections to show dependencies of technology to other technology could be introduced. 
+
+**Primer per category** 
+A primer template has been created and an example AI Primer can be found in the resources. A primer can be utilised to further explain the categories at the top of the landscape in relation to what it is, examples of it and other details listed in the template.
+
+**Disruption**
+Work into understanding how disruptive the technology listed in the landscape is to the current environment should be investigated. This should be visualised on the landscape itself, and integrating disruption into the Primer. 
+
+
+*Author
```

Comment:
Make this 1 line instead of 2, just b'cos it doesn't recognise the 2 anyway :stuck_out_tongue:

Reply:
Or rather, your name appears as the author at the top of the post already anyway so maybe you can remove it?